Fix the commands for image build in the docs and script.

aztecher opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

All changes must adhere to this template to make it easy for reviewers
and preserve rationale/history behind every change
-->

## 📑 Description
<!-- Add a brief description of the pr -->

<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

It seems that the build commands for container images of ubuntu and fedora were changed in the https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4899, but this did not seem to be reflected in the docs and script, so fix it.

## Additional Information for reviewers
<!--
What exactly did you change - you may also defer to information
contained in commit messages. At a bare minimum it's worth highlighting
which areas of the code were changed as it's easier to assign reviewers
-->

## ✅ Checks
<!-- Make sure your pr passes the CI checks and do check the following fields as needed - -->
- [ ] My code requires changes to the documentation
- [ ] if so, I have updated the documentation as required
- [ ] My code requires tests
- [ ] if so, I have added and/or updated the tests as required
- [ ] All the tests have passed in the CI <!-- If not leave a comment as to why the CI is red and if you need help understanding what's wrong -->

## How to verify it
<!--
Did you include unit tests? or end-to-end tests?
How can I manually verify that this patch achieves its objective
-->

aztecher commented:
/retest

flavio-fernandes reviewed (approved):
nice catch. TY @aztecher

flavio-fernandes reviewed (changes requested):
@aztecher Could you also please fix the following doc:

https://github.com/ovn-kubernetes/ovn-kubernetes/blame/bb78ce88b1eed866c46e40abd5839856601cf064/docs/observability/ovn-observability.md#L48


Apparently this should be changed to:

`OVS_BRANCH=main make -C dist/images fedora-dev-local-gw-deployment` ?!?

@qinqon  @npinaeva  ^^

And thank you again for this PR!

npinaeva commented:
that is interesting naming indeed (:

aztecher commented:
@flavio-fernandes 
Thank you for your review! 
I've addressed your comment in the latest commit.

Please let me know if anything else needs tweaking ;)

aztecher commented:
/retest

flavio-fernandes reviewed (approved):
/lgtm
